Header header
uint32 ID
uint32 imageAmount
uint32 imageDelay
uint32 boardWidth
uint32 boardHeight
float32 boardRectangleWidth
float32 boardRectangleHeight